packages/node/src/indexer/fetch.service.spec.ts

```diff
@@ -126,31 +127,39 @@ describe('FetchService', () => {
   it('skip load more if has preloaded enough', () => {
     const apiService = mockApiService();
     const batchSize = 50; // preload = 50*2 = 100
+    const dictionaryService = new DictionaryService();
+    const project = testSubqueryProject();
     const fetchService = new FetchService(
       apiService,
       new NodeConfig({ subquery: '', subqueryName: '', batchSize }),
+      project,
+      dictionaryService,
       new EventEmitter2(),
     );
     (fetchService as any).latestFinalizedHeight = 1000;
-    (fetchService as any).latestPreparedHeight = 200;
+    (fetchService as any).latestBufferedHeight = 200;
     (fetchService as any).latestProcessedHeight = 100;
     expect((fetchService as any).nextBlockRange(0)).toBeUndefined();
   });
 
   it('skip load more if prepared height >= finalized height', () => {
     const apiService = mockApiService();
     const batchSize = 50; // preload = 50*2 = 100
+    const dictionaryService = new DictionaryService();
+    const project = testSubqueryProject();
     const fetchService = new FetchService(
       apiService,
       new NodeConfig({ subquery: '', subqueryName: '', batchSize }),
+      project,
+      dictionaryService,
       new EventEmitter2(),
     );
     (fetchService as any).latestFinalizedHeight = 1000;
-    (fetchService as any).latestPreparedHeight = undefined;
+    (fetchService as any).latestBufferedHeight = undefined;
     (fetchService as any).latestProcessedHeight = undefined;
     expect((fetchService as any).nextBlockRange(1001)).toBeUndefined();
```

Comment:
I think now `nextBlockRange` won't change latestBufferedHeight and latestProcessedHeight